gwpy/timeseries/tests/test_timeseries.py

```diff
@@ -1388,6 +1388,14 @@ def test_boolean_statetimeseries(self, array):
         assert comp.name == '%s >= 2.0' % (array.name)
         assert (array == array).name == '{0} == {0}'.format(array.name)
 
+    @pytest_skip_network_error
+    def test_transferfunction(self):
+        tsh = TimeSeries.fetch_open_data('H1', 1126259446, 1126259478)
+        tsl = TimeSeries.fetch_open_data('L1', 1126259446, 1126259478)
+        tf = tsh.transferfunction(tsl, fftlength=1.0, overlap=0.5)
+        assert tf.df == 1 * units.Hz
+        assert tf.frequencies[abs(tf).argmax()] == 516 * units.Hz
+
```

Comment:
Is it possible to define an end-to-end test that doesn't rely on remote data? i.e. with some noise created using `numpy` or similar?

If that's not trivial, don't worry about it, the `@pytest_skip_network_error` decorator is enough.

Reply:
I'll note that `test_coherence()` is basically identical. in that it fetches remote data. If you desire no remote fetching and instead create some noise with `numpy`, I'll need to seed the RNG. While not trivial and can be done, there are plenty of other tests in this file which are fetching open data. Perhaps a different issue or PR can be created that addresses most/all of the fetches of remote data.

Reply:
@duncanmmacleod Can I resolve this conversation or do you want to iterate further?

Reply:
Ping @duncanmmacleod. Can this be resolved since this fetch of open data appears many other places?

Reply:
Having not heard anything further, I'll mark this as resolved.